### ISSUE TITLE
StitchingBuilder

### ISSUE THREAD

**Issue by michaelstaib** [ChilliCream/hotchocolate#568]
# Builder Support

For the stitched schema we should provide a builder class that makes it easy to create the stitched layer.

```csharp
StichingBuilder.New()
    .AddRemoteSchema(foo)
    .AddExtensionFromFile("Extensions.graphql")
    .AddConflictResolver(types => new ConflictResolution(false))
    .AddFieldSelector((type, field) => false)
    .RenameField("schemaName", new FieldReference("Type", "field"), "newFieldName")
    .Merge();
```

**Comment by MakcStudio** [comment]
how its work?

**Comment by michaelstaib** [comment]
The stitching builder is still in progress ... we plan to release it at the end of next week.

**Comment by michaelstaib** [comment]
The stitching builder is now implemented and included with 0.8.0-preview.3
